Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM node:8.9.1-alpine
 
-# RUN apk add --no-cache python=2.7.13-r1 git-perl bash make gcc g++
-# RUN rm /bin/sh && ln -s /bin/bash /bin/sh
+RUN apk add --no-cache python==2.7.14-r0 git-perl bash make gcc g++
```

Comment:
Is there supposed to be a double equals here?

Reply:
no, good spot.